### ISSUE TITLE
WCAG Searchbar issue

### ISSUE THREAD

**Issue by eal172** [yext/search-ui-react#355]
The default searchbar is causing a wcag error when used in the locator because of a "Missing form label"

Link to example site: https://v10trunk-local-bumpertobumper-ca.preview.pagescdn.com/en/search
